src/Provider/ImageProvider.php

```diff
@@ -88,7 +90,8 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
         $params = [
             'alt' => $media->getName(),
             'title' => $media->getName(),
-            'src' => $this->generatePublicUrl($media, $format),
+            'src' => '#',
```

Comment:
Not sure if this is even better... Is this a valid `src` url?

Reply:
Looks like it is. Paste this:

```
<!DOCTYPE html>
<html lang="en">
<head><title>test</title></head>
<body>
<img src="#" alt="">
</body>
</html>
```

here: https://validator.w3.org/#validate_by_input

Reply:
@core23 what do you think?